README.md

```diff
@@ -87,6 +87,14 @@ output-artifact: typescript-files
 ### REST Client Generator overrides
 
 ```yaml $(rest-level-client)
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+source-code-folder-path: ./src

```

Comment:
I don't think that's applicable for multi-client ?

Reply:
When adding multi client support we would probably need to add another check for the multi-client flag and setup the right directives for this.  Such as `$(rest-level-client) && !$(multi-client)`

Reply:
For multi-client, we can define the `source-code-folder-path` in the README.md which is stored in sdk repo. Then the `source-code-folder-path` will be overrided.